protocols/gossipsub/src/behaviour.rs

```diff
@@ -2847,164 +2743,28 @@ where
     /// Takes each control action mapping and turns it into a message
     fn flush_control_pool(&mut self) {
```

Comment:
This probably exceeds the scope of this refactoring but I am curious to understand if the holding back of these messages is that important?

If we are now adding prioritization to the sending of messages, would it make sense to remove this pool and instead send them with a lower priority?

Reply:
yeah makes sense, can we leave this to the prioritization PR?